tests/integration/test_api_gateway.py

```diff
@@ -127,3 +148,40 @@ def forward_request(self, **kwargs):
 
     # clean up
     proxy.stop()
+
+
+# def test_api_gateway_lambda_proxy_integration():
```

Comment:
What is the plan for this test method - I'm assuming this is still work in progress on your end?